### PR TITLE
New station does not remove historic station

### PR DESCRIPTION
**Pull request by johan-t**
# Pull Request Template

## Description

There was an issue where when you added a new station one of the historic stations was removed because we would have enough real data to fill the map. I solved the issue by instead of overwriting the state with the new data we are instead appending the new data to the current state. Meaning if a station would not be returned from the api because we have enough data it would still be displayed for the user that has reported the station.

Fixes: #95 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

tested around a bit 

## Checklist:

Before submitting your PR, please review the following checklist:

- [x] I have performed a self-review of my own code or materials.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.
- [x] Any dependent changes have been merged and published in downstream modules.

## Additional Information:

Any additional information that you believe is important and relevant to the review of this PR.

### CODE REVIEW

**Comment by johan-t** [comment]
@brandesdavid I forgot but is this safe to merge?

**Comment by brandesdavid** [comment]
@johan-t yes it is, tried to add something, but it is working

**Comment by brandesdavid** [comment]
Important!! New report doesn't delete the old historisch Station, this means historische data stays

**Comment by brandesdavid** [comment]
@johan-t please test the changes. i changed the variable name x to duplicate station, and added and if statement which checks if the current station timestamp is greater or is not historic.

**Comment by johan-t** [comment]
@brandesdavid there is something strange happening when reporting a new station the station that was previously reported is getting set again to 1 and then after 5 seconds set to the correct opacity. Something strange that I have noticed not a big issue because most people dont report 2 stations so we could merge imo but what do you think?

File: src/components/Map/Markers/MarkerContainer.tsx [hunk 1]
```diff
@@ -4,66 +4,78 @@ import { getRecentTicketInspectorInfo } from '../../../utils/dbUtils';
...
-			// Check if the new array is not empty, then update the state
-			if (Array.isArray(newTicketInspectorList) && newTicketInspectorList.length > 0) {
+            // Append new data if the new array is not empty
+            if (Array.isArray(newTicketInspectorList) && newTicketInspectorList.length > 0) {
+                // Creating a Set to remove duplicates based on station id
+                const updatedList = [...ticketInspectorList, ...newTicketInspectorList].reduce((acc: MarkerData[], current: MarkerData) => {
+                    const duplicateStation = acc.find(item => item.station.id === current.station.id);
+                    if (!duplicateStation) {
+                        return acc.concat([current]);
+                    } else {
```
Comment (johan-t) [inline src/components/Map/Markers/MarkerContainer.tsx:48]: can you turn this into an else if statement instead?

**Comment by brandesdavid** [comment]
changed the if station from duplicateStation inside MarkerContainer.tsx to an else if statement
@johan-t  please review

**Pull request closed without merging**